csrc/scheduler/matmul_heuristic_plugin.cpp

```diff
@@ -161,11 +161,10 @@ void copyConfigToParams(MatmulParams* mparams, const KernelConfig* config) {
   };
   setGemmTile(mparams->tile_sizes.cta_tile, config->cta_tile);
   setGemmTile(mparams->tile_sizes.warp_tile, config->warp_tile);
-  setGemmTile(mparams->tile_sizes.instruction_tile, config->instruction_tile);
   mparams->circular_buffer_options.smem_circular_buffer_stage =
       config->load_stages;
   mparams->async_gmem_load_operands = config->async_gmem_load_operands;
-  // Update mma macro if necessary to match instruction tile
+  // Update mma macro if necessary to match provided instruction tile
```

Comment:
Can the `mma_macro` conflict with `config->warp_tile` for hopper?

Reply:
In either case the warp tile must be a multiple of instruction tile, but we don't do any explicit checking in the plugin. The checks come once we start the scheduling.

Reply:
Note we don't need to enforce warp_tile==instruction_tile on Hopper: https://github.com/NVIDIA/Fuser/pull/3278#discussion_r1834897133. I was doing that because I thought we needed it but there are configs that can use multiple instructions per warp.